packages/table-ui/package.json

```diff
@@ -30,14 +30,19 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "slate": "^0.60.0",
-    "slate-react": "^0.60.0"
+    "slate-react": "^0.60.0",
+    "slate-history": "^0.62.0",
+    "@udecode/plate-common": "^2.0.0",
+    "styled-components": ">=5.0.0"
```

Comment:
Why is this still here?

Reply:
@roosnic1 What do you suggest? move it to dependencies? or leave it and add it to dependencies in core package?

Reply:
If `styled-components` is needed then add it to the dependencies. I don't understand why you would want to add it to the peerDependencies

Reply:
same as slate and react, they are both needed but will always be there, that's why I suggest we add styled-components in core as we use it in multiple plugins and plate talked about adding it [here](https://plate.udecode.io/docs/installation)

Reply:
resolved 👍